README.md

```diff
@@ -1,2 +1,24 @@
-# operator-pipelines
-OpenShift Pipelines for Partner Operator Bundle certification
+# Operator pipelines
+
+## Operator CI pipeline
+
+Operator CI pipeline is a pipeline that can be triggered by partner on on-premise
+infrastructure. The pipeline does basic check of new operator, build it and install
+it in ocp environment. After an operator is installed a pre-flight tests are executed
+that validates that operator meets minimum requirements for Red Hat OpenShift Certification.
+If tests pass a CI pipeline submits a PR for full operator certification workflow.
+
+To trigger a CI pipeline follow steps bellow:
+
+```bash
+oc apply -R -f pipelines
+oc apply -R -f tasks
```

Comment:
It'd be nice if the install steps for each pipeline included only what is absolutely required so as not to pollute the partner's cluster with useless resources. Perhaps the directory structure needs further consideration or install/uninstall scripts are warranted. This is just something to consider for now. I'll file something in the backlog for it.